pkg/spoke/submarineragent/gateways_controller.go

```diff
@@ -21,6 +21,7 @@ import (
 
 const (
 	submarinerGatewayLabel        = "submariner.io/gateway"
+	GatewayLabelledBySubmariner   = "submariner.io/random-gateway-node"
```

Comment:
This can be private.

Reply:
Also one 'l':

```suggestion
	gatewayLabeledBySubmariner   = "submariner.io/random-gateway-node"
```

Reply:
That’s a UK/US variation, either is fine by me.